store/src/server/api.rs

```diff
@@ -196,24 +196,22 @@ impl api_server::Api for StoreApi {
         Ok(Response::new(ListNullifiersResponse { nullifiers }))
     }
 
-    // Returns a list of all accounts
-    async fn list_accounts(
+    // Returns a list of all notes
```

Comment:
minor nit: here, and in other similar places, we should use doc-comments (i.e., `///`).